types/stopword/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for stopword 0.3
+// Project: https://github.com/fergiemcdowall/stopword
+// Definitions by: Rico Sandyca Novenza <https://github.com/ricosandyca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace Stopword {
```

Comment:
Rather than using `declare namespace` and `export =`, this entire definition could be written more simply using regular top-level ES module export syntax:

```ts
export function removeStopwords(...)
export const af: string[];
export const ar: string[];
...
```

Also, according to the documentation, the `stopword` package can also be used as a script in the browser as the identifier `sw`, so I would recommend adding an `export as namespace sw;` to this file to support that use case.